lib/sufia.rb

```diff
@@ -41,4 +41,8 @@ def self.config(&block)
 
     @config
   end
+
+  def self.primary_work_type
+    CurationConcerns::WorkRelation::DummyModel.primary_concern
```

Comment:
If there are multiple concerns, this would be the first one listed in the config file?